platform/src/test/java/org/exoplatform/platform/qa/ui/platform/social/SOCHomePageTestIT.java

```diff
@@ -0,0 +1,885 @@
+   * <li>Pre-Condition:</li>
+   * <li>Post-Condition:</li> Step Number: 1 Step Name: - Add a link Step
+   * Description: - Log in - Goto Homepage - In Activity Composer click Add Link -
+   * Input URL and click Attach Input Data: Expected Outcome: - A input text field
+   * shows up to users to input the URL - The infomation of URL is shown in
+   * featured content parts click(hp.ELEMENT_PUBLICATION_ADDLINK);
+   * type(hp.ELEMENT_PUBLICATION_ADDLINK_INPUT, "http://www.ted.com", false);
+   * click(hp.ELEMENT_PUBLICATION_ADDLINK_ATTCHBTN); Step number: 2 Step Name: -
+   * Share the document Step Description: - Click Share button Input Data:
+   * Expected Outcome: - Activity is added into activity stream
+   */
+  @Test
+  public void test08_AddALink() {
+    info("Test 8: Add a link");
+    String textDes = "textDes" + getRandomNumber();
+    String link = "http://www.google.fr";
+
+    homePagePlatform.goToHomePage();
+    activityStream.addActivity(textDes, link);
+    $(byText(link)).should(Condition.exist);
+    $(byText(textDes)).should(Condition.exist);
+    executeJavaScript("window.scrollBy(0,-1500)", "");
+    $(byText(link)).parent()
+                   .parent()
+                   .parent()
+                   .parent()
+                   .parent()
+                   .find(byClassName(ELEMENT_DATE_ACTIVITY))
+                   .waitUntil(Condition.visible, Configuration.timeout)
+                   .hover();
+    $(byText(link)).parent().parent().parent().parent().parent().find(ELEMENT_ICON_DELETE_ACTIVITY).click();
+    ELEMENT_DELETE_POPUP_OK.waitUntil(Condition.visible, Configuration.timeout).click();
+    $(byText(link)).parent().parent().parent().parent().parent().waitUntil(Condition.disappear, Configuration.timeout);
+
+  }
+
+  /**
+   * <li>Case ID:121927.</li>
+   * <li>Test Case Name: Load previous activity page automatically.</li>
+   * <li>Pre-Condition:</li>
+   * <li>Post-Condition:</li> Step Number: 1 Step Name: - Create more than 1 page
+   * of activities (Default 20) Step Description: - Log in - Go to homepage and
+   * create more than 20 activities Input Data: Expected Outcome: - more than 20
+   * activities are created successfully Step number: 2 Step Name: - check the
+   * loading automaticallyprevious activity page Step Description: - log out
+   * andlog in again - go to home page - scroll to the bottom of the activity
+   * stream Input Data: Expected Outcome: -the first page of last activities is
+   * displayed - previous activities' pages are load automatically
+   */
+  @Test
+  public void test09_LoadPreviousActivityPageAutomatically() {
+    info("Test 9: Load previous activity page automatically");
+    String textDesfirst = "textDesfirst" + getRandomNumber();
+    activityStream.addActivity(textDesfirst, "");
+    refresh();
+    for (int i = 0; i <= 15; i++) {
+      String act = "act" + getRandomNumber();
+      activityStream.addActivity(act, "");
+      refresh();
```

Comment:
Do we really need to refresh at each iteration ?

Reply:
refresh removed